src/builder.rs

```diff
@@ -27,7 +31,7 @@ impl<K: EnrKey> Default for Builder<K> {
     /// Constructs a minimal [`Builder`] for the v4 identity scheme.
     fn default() -> Self {
         Self {
-            id: String::from("v4"),
+            id: String::from_utf8(ENR_VERSION.into()).expect("A valid constant"),
```

Comment:
```suggestion
            id: String::from_utf8_unchecked(ENR_VERSION.into()),
```

Reply:
This is `unsafe`, and rather than allowing unsafe rust code, i think the `expect()` is better.

Reply:
right sorry, missed the `unsafe`